signing_service/signing_service/signing_service.py

```diff
@@ -164,6 +177,8 @@ def _parse_arguments() -> argparse.Namespace:
 
     arg_host = args.concent_cluster_host
     arg_port = args.concent_cluster_port
+    arg_concent_public_key, _ = escape_decode(args.concent_public_key)
```

Comment:
Please add a proper name for the second argument (can still start with `_` but it should have a name) and put parentheses around it (it's a tuple).

Reply:
:+1: done